modules/viewing-archived-pipeline-runs.adoc

```diff
@@ -20,12 +20,13 @@ endif::[]
 * An archived pipeline run exists.
 
 .Procedure
-. From the {productname-short} dashboard, click *Data Science Pipelines* -> *Runs*.
-. On the *Runs* page, select the *project* for the archived pipeline runs that you want to view.
-. Click the *Archived* tab.
+. From the {productname-short} dashboard, click *Experiments* -> *Experiments and runs*.
+. On the *Experiments* page, select the *project* that contains the pipeline experiment for the archived pipeline runs that you want to view.
+. From the list of pipeline experiments, click the pipeline experiment that contains the archived pipeline runs that you want to view.
```

Comment:
Can omit the second 'pipeline' here.

Reply:
Omitted the second occurrence of 'pipeline'.